lib/matplotlib/patches.py

```diff
@@ -571,6 +572,14 @@ def get_hatch(self):
         """Return the hatching pattern."""
         return self._hatch
 
+    def set_hatch_linewidth(self, hatch_linewidth):
+        """Set the hatch linewidth"""
+        self._hatch_linewidth = hatch_linewidth
+
+    def get_hatch_linewidth(self):
+        """Return the hatch linewidth"""
+        return self._hatch_linewidth
```

Comment:
Probably should be consistent with the `Collection` version:
```suggestion
    def set_hatch_linewidth(self, lw):
        """Set the hatch linewidth."""
        self._hatch_linewidth = lw

    def get_hatch_linewidth(self):
        """Return the hatch linewidth."""
        return self._hatch_linewidth
```